service/pixelated/config/site.py

```diff
@@ -2,12 +2,15 @@
 
 
 class AddCSPHeaderRequest(Request):
```

Comment:
I think we should rename this Class, as by now it no longer only adds CSP headers. How about AddSecurityHeadersRequest?